wgpu-hal/src/dx12/adapter.rs

```diff
@@ -174,7 +174,8 @@ impl super::Adapter {
             | wgt::Features::MULTI_DRAW_INDIRECT
             | wgt::Features::MULTI_DRAW_INDIRECT_COUNT
             | wgt::Features::ADDRESS_MODE_CLAMP_TO_BORDER
-            | wgt::Features::NON_FILL_POLYGON_MODE
+            | wgt::Features::LINE_POLYGON_MODE
```

Comment:
It would be more consistent (e.g. with "ADDRESS_MODE_CLAMP_TO_BORDER") to call it the other way around: "POLYGON_MODE_LINE" and "POLYGON_MODE_POINT"